packages/x-date-pickers/src/CalendarPicker/CalendarPicker.tsx

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import clsx from 'clsx';
 import { styled, useThemeProps } from '@mui/material/styles';
 import { unstable_composeClasses as composeClasses } from '@mui/material';
+import useId from '@mui/utils/useId';
```

Comment:
Usually we don't use the absolute import.

```suggestion
import { unstable_useId as useId } from '@mui/utils';
```

Reply:
Thanks for noting it. 👍 
I see that almost every other instance imports this util from `@mui/material/utils` rather than `@mui/utils`, I'll replace the one existence of `@mui/utils` with the former. 😉